prism_compile.c

```diff
@@ -4699,18 +4699,25 @@ pm_compile_node(rb_iseq_t *iseq, const pm_node_t *node, LINK_ANCHOR *const ret,
                   // def foo(a, (b, *c, d), e = 1, *f, g, (h, *i, j),  k:, l: 1, **m, &n)
                   //                                                             ^^^
                   case PM_KEYWORD_REST_PARAMETER_NODE: {
+                        pm_keyword_rest_parameter_node_t *kw_rest_node = (pm_keyword_rest_parameter_node_t *)parameters_node->keyword_rest;
                         if (!body->param.flags.has_kw) {
                             body->param.keyword = keyword = ZALLOC_N(struct rb_iseq_param_keyword, 1);
                         }
 
                         keyword->rest_start = local_index;
                         body->param.flags.has_kwrest = true;
 
-                        pm_constant_id_t constant_id = ((pm_keyword_rest_parameter_node_t *)parameters_node->keyword_rest)->name;
+                        pm_constant_id_t constant_id = kw_rest_node->name;
                         if (constant_id) {
                             pm_insert_local_index(constant_id, local_index, index_lookup_table, local_table_for_iseq, scope_node);
-                            local_index++;
                         }
+                        else {
+                            const ID kw_splat_id = parse_location_symbol(&kw_rest_node->operator_loc, parser);
```

Comment:
I think this can be `ID kw_splat_id = idPow;`. It's very hard to find, but there are a bunch of builtin IDs in `defs/id.def`